features/Context/FieldTypeFormContext.php

```diff
@@ -144,23 +145,120 @@ public function theFieldDefinitionIsMarkedAsRequired()
      */
     public function theInputFieldsShouldBeFlaggedAsRequired()
     {
-        $inputNodeElements = $this->getSession()->getPage()->findAll(
+        $this->theFieldShouldBeFlaggedAsRequired('input');
+    }
+
+    /**
+     * @Given /^the field definition is set to single choice$/
```

Comment:
You could go for `/^the field definition is set to :value choice$/`, with a method argument, and handle single/multiple in the method. It would allow you to remove one. Just unsure if it is worth it or not :)